pycbc/types/timeseries.py

```diff
@@ -84,6 +83,13 @@ def __init__(self, initial_array, delta_t=None, epoch="", dtype=None, copy=True)
         self._delta_t = delta_t
         self._epoch = epoch
 
+    def epoch_close(self, other):
+        """ Check if the epoch is close enough to allow operations """
+        dt = abs(float(self.start_time) - float(other.start_time))
```

Comment:
Considering cases where `start_time` is not a float (e.g. `LIGOTimeGPS`), is it better to subtract first and convert to float later? I suspect it will not make a big difference given the tolerance on `dt`, but I am wondering if you thought about it.

Reply:
I haven't, I'll make the minor change just in case. Longer term, we should think about if we can abandon LIGOTimeGPS altogether. I've found some cases where it is kind of a pain. Not a high priority though.